internal/resources/syntheticmonitoring/resource_check.go

```diff
@@ -961,6 +1141,95 @@ func makeCheckSettings(settings map[string]interface{}) sm.CheckSettings {
 		}
 	}
 
+	multihttp := settings["multihttp"].(*schema.Set).List()
+	if len(traceroute) > 0 {
+		m := multihttp[0].(map[string]interface{})
+
+		cs.Multihttp = &sm.MultiHttpSettings{}
+
+		if m["entries"].(*schema.Set).Len() > 0 {
```

Comment:
This whole section might be overkill? I'm not really grokking how this exactly works